### PR TITLE
Upgrade build-tools version to 4.0.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by jimma**

### CODE REVIEW

**Review by reta — APPROVED** [review]

**Comment by reta** [comment]
Thanks @jimma !

File: pom.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
     </prerequisites>
 
     <properties>
-        <cxf-buildtools-version>3.4.5-SNAPSHOT</cxf-buildtools-version>
+        <cxf-buildtools-version>4.0.0-SNAPSHOT</cxf-buildtools-version>
         <jdk.version>1.8</jdk.version>
```
Comment (coheigea) [inline pom.xml:48]: Shouldn't the JDK be bumped as well?

**Review by coheigea — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (reta) [reply pom.xml:48]: @coheigea good one, it should, I could do this in next pull request

Comment (coheigea) [reply pom.xml:48]: OK sounds good @reta

**Pull request merged**